src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,118 @@
+package mate.jdbc.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.custom.Exception;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.manufacturer.Manufacturer;
+import mate.jdbc.util.UtilClass;
+
+@Dao
+public class ManufacturerDaoImpl implements ManufacturerDao {
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        try (Connection connection = UtilClass.getConnection();
+                 PreparedStatement createdManufacturerStatement = connection
+                         .prepareStatement("INSERT INTO manufacturers(name, country) values(?, ?);",
+                         Statement.RETURN_GENERATED_KEYS)) {
+            createdManufacturerStatement.setString(1, manufacturer.getName());
+            createdManufacturerStatement.setString(2, manufacturer.getCountry());
+            createdManufacturerStatement.executeUpdate();
+            ResultSet generatedKey = createdManufacturerStatement.getGeneratedKeys();
+            if (generatedKey.next()) {
+                Long id = generatedKey.getObject(1, Long.class);
+                manufacturer.setId(id);
+            }
+            return manufacturer;
+        } catch (SQLException e) {
+            throw new Exception("Can't create manufactures in DB taxi_db", e);
```

Comment:
просмотри все exception месседжи и добавь детали

```suggestion
            throw new Exception("Can't create manufactures in DB: " + manufacturer, e);
```